pkg/kapp/clusterapply/add_or_update_change.go

```diff
@@ -235,11 +236,19 @@ func (c AddOrUpdateChange) recordAppliedResource(savedRes ctlres.Resource) error
 		}
 
 		// Record last applied change on the latest version of a resource
-		latestResWithHistoryUpdated, err := latestResWithHistory.RecordLastAppliedResource(applyChange)
+		latestResWithHistoryUpdated, isAnnValueMaxLengthExceeded, err := latestResWithHistory.RecordLastAppliedResource(applyChange)
```

Comment:
isAnnValueMaxLengthExceeded => madeAnyModifications

we should make this transparent to consumers why we have not make any modifications.

Reply:
Sure. I have started checking the negation of `madeAnyModifications ` as I think it makes more sense and reverted all the bool values.